### PR TITLE
[PS-2099] PostgreSQL EF Contains() in SQL Query is strpos(), witch is case-sens…

### PR DESCRIPTION
**Pull request by juliokele**
After Items importing the Cipher/Folders value is uppercase.
PostgreSQL EF Contains() under the hood in SQL Query is strpos(), witch is case-sensitive.

### CODE REVIEW

**Comment by justindbaur** [comment]
Hi @juliokele thank you so much for this contribution. Did you by chance test this against MySQL to make sure this solution still allowed it to work properly?

**Comment by juliokele** [comment]
I have tested against MySQL and working properly.

File: src/Infrastructure.EntityFramework/Repositories/Queries/CipherDetailsQuery.cs [hunk 1]
```diff
@@ -28,7 +28,7 @@ public virtual IQueryable<CipherDetails> Run(DatabaseContext dbContext)
                         DeletedDate = c.DeletedDate,
                         Reprompt = c.Reprompt,
                         Favorite = _userId.HasValue && c.Favorites != null && c.Favorites.Contains($"\"{_userId}\":true"),
```
Comment (kspearrin) [inline src/Infrastructure.EntityFramework/Repositories/Queries/CipherDetailsQuery.cs:30]: Dont we have this same problem here too? I think we do these kinds of checks in some other areas too. @justindbaur `UserCipherDetailsQuery` maybe?

File: src/Infrastructure.EntityFramework/Repositories/Queries/CipherDetailsQuery.cs [hunk 2]
```diff
@@ -28,7 +28,7 @@ public virtual IQueryable<CipherDetails> Run(DatabaseContext dbContext)
...
-                        FolderId = (_ignoreFolders || !_userId.HasValue || c.Folders == null || !c.Folders.Contains(_userId.Value.ToString())) ?
+                        FolderId = (_ignoreFolders || !_userId.HasValue || c.Folders == null || !c.Folders.ToLowerInvariant().Contains(_userId.Value.ToString())) ?
```
Comment (kspearrin) [inline src/Infrastructure.EntityFramework/Repositories/Queries/CipherDetailsQuery.cs:31]: Do we need to ensure `_userId` is lowercase as well?

File: src/Infrastructure.EntityFramework/Repositories/Queries/CipherDetailsQuery.cs (hunk 1 above)
Comment (juliokele) [reply src/Infrastructure.EntityFramework/Repositories/Queries/CipherDetailsQuery.cs:30]: yes, the same problem (Favorite is upper-case too):
https://github.com/bitwarden/server/blob/481ce38934922222c4d4054aed9c781583895723/src/Core/Services/Implementations/CipherService.cs#L645

**Review by kspearrin — APPROVED** [review]

**Review by kspearrin — APPROVED** [review]

**Comment by juliokele** [comment]
@kspearrin
Sorry for deleting my head repository before PR merging!

**Comment by kspearrin** [comment]
@juliokele no worries, we're trying to figure out some issues with unsigned commits before being able to merge.

**Comment by kspearrin** [comment]
@justindbaur Should we merge this for next release?

**Comment by justindbaur** [comment]
@kspearrin Yes we should, have we figured out the unsigned commit issue?

**Pull request merged**